src/platform/site-wide/maintenance-banner/config.js

```diff
@@ -0,0 +1,22 @@
+// Node modules.
+import moment from 'moment';
+
+// Derive startsAt and expiresAt.
+const startsAt = moment.utc('2020-06-02T17:00:00.248Z').local();
+const expiresAt = moment.utc('2020-06-03T19:00:00.248Z').local();
+
+// Format startsAt and expiresAt.
+const formattedStartsAt = startsAt.format('dddd M/D, h:mm a');
+const formattedExpiresAt = expiresAt.format('dddd M/D, h:mm a');
+
+export default {
+  id: '1',
+  startsAt,
+  expiresAt,
+  title: 'DS Logon is down for maintenance.',
+  content:
+    'DS Logon is down for maintenance. Please use ID.me or MyHealtheVet to sign in or use online tools.',
+  warnStartsAt: startsAt.clone().subtract(12, 'hours'),
+  warnTitle: 'DS Logon will be down for maintenance',
+  warnContent: `DS Logon will be unavailable from ${formattedStartsAt} to ${formattedExpiresAt} Please use ID.me or MyHealtheVet to sign in or use online tools during this time.`,
+};
```

Comment:
I think the title should always be `System maintenance`.

I'm torn on the content sections because on one hand, the content is different on warn vs. active downtime and full downtime vs DS Logon downtime, but also I think it should be standard based on site-wide vs. service-level. 🤔

Reply:
I actually just grabbed the same content that @peggygannon created for the original PrePreDowntime and Downtime components we had in announcements 🙂 I do think the content will be updated again once we enable downtime messaging at some point in the future